pluginconfig/token_test.go

```diff
@@ -84,19 +94,24 @@ func Test_TokenDataObserver_Unmarshall(t *testing.T) {
 					Type:    "usdc-cctp",
 					Version: "1.0",
 					USDCCCTPObserverConfig: &USDCCCTPObserverConfig{
+						AttestationConfig: AttestationConfig{
+							AttestationAPI:         "http://localhost:8080",
+							AttestationAPITimeout:  commonconfig.MustNewDuration(time.Second),
+							AttestationAPIInterval: commonconfig.MustNewDuration(500 * time.Millisecond),
+							AttestationAPICooldown: commonconfig.MustNewDuration(10 * time.Minute),
```

Comment:
Do we have a cooldown for Lombard API?

Reply:
I mean, if you plan to use AttestationConfig across different token providers (for instance Lombard) and Lombard doesn't need cooldown then what's the point of having that field in this generic struct? If this AttestationConfig is meant to be only for USDC then it should be fine.

I guess we can refine it in the PR in which we add lbtc support